app/services/decisions/self_employed_income_decision_tree.rb

```diff
@@ -17,7 +17,13 @@ def destination # rubocop:disable Metrics/MethodLength, Metrics/CyclomaticComple
       when :business_employees
         edit(:business_financials, business_id:)
       when :business_financials
-        edit(:businesses_summary, business_id:)
+        after_business_financials
+      when :business_salary_or_remuneration
+        edit(:business_total_income_share_sales, business_id: form_object.record)
```

Comment:
business_id is set as a method below so form_object.record is not required

Reply:
how did i miss that 🤦‍♀️